### PR TITLE
DISC-521 implemented

### PR DESCRIPTION
**Pull request by thomasegense**
Very straightforward use of streaming client.
Added optional mTime parametere to solrIndex method.

### CODE REVIEW

File: src/main/openapi/ds-datahandler-openapi_v1.yaml [hunk 1]
```diff
@@ -108,19 +108,27 @@ paths:
                 
                
                 
-  /solr/index/full:
+  /solr/index:
```
Comment (tokee) [inline src/main/openapi/ds-datahandler-openapi_v1.yaml:111]: Fitting rename for now. When we introduce iterative updates matching `/oai/import/{full|update}`, we can rename it again.

File: src/main/java/dk/kb/datahandler/facade/DsDatahandlerFacade.java [hunk 2]
```diff
@@ -94,57 +96,53 @@ public static ArrayList<String> ingestFromZipfile(String origin, InputStream is)
...
-            log.error("Error calling ds-present or solr.",e);
-            throw new InternalServiceException("Error calling ds-present or solr");
-        }
-                      
+            .setParameter("commit", "true")
+            .build().toURL();      
      
+                
+        boolean hasMore=true;        
+        long batchSize=100L; //There seems to no performance boost increasing to 1000. Better keep in small batches.
```
Comment (tokee) [inline src/main/java/dk/kb/datahandler/facade/DsDatahandlerFacade.java:122]: Currently we have too few records to really measure and in the future we might ass speech-to-text and get quite large records. Please make this a property (with default as you choose).

**Review by tokee — CHANGES_REQUESTED** [review]
Very nice. Only suggestion is to make the batch size a property.

File: src/main/java/dk/kb/datahandler/facade/DsDatahandlerFacade.java (hunk 2 above)
Comment (thomasegense) [reply src/main/java/dk/kb/datahandler/facade/DsDatahandlerFacade.java:122]: Added

File: src/main/java/dk/kb/datahandler/config/ServiceConfig.java [hunk 3]
```diff
@@ -49,6 +49,7 @@ public static synchronized void initialize(String configFile) throws IOException
         oaiTimestampFolder= serviceConfig.getString("config.timestamps.folder");
         dsStorageUrl = serviceConfig.getString("config.storage.url");
         solrUrl = serviceConfig.getString("config.solr.url");
+        solrBatchSize=  serviceConfig.getInteger("config.solr.batchSize");
```
Comment (tokee) [inline src/main/java/dk/kb/datahandler/config/ServiceConfig.java:52]: For low priority things like this, please add a default value to the `getInteger` call. I'll accept the pull request as one is given in the `ds-datahandler-behaviour.yaml`.

**Review by tokee — APPROVED** [review]

**Pull request merged**